1/contrib/s2i/run

```diff
@@ -12,6 +12,24 @@ source /usr/local/bin/kube-slave-common.sh
 jenkins_config_dir="/opt/openshift/configuration"
 jenkins_config_path="${jenkins_config_dir}/config.xml"
 
+
+CONTAINER_MEMORY_IN_BYTES=`cat /sys/fs/cgroup/memory/memory.limit_in_bytes`
+echo "Container memory in bytes ${CONTAINER_MEMORY_IN_BYTES}"
+DEFAULT_MEMORY_CEILING=$((2**30-1))
+echo "Default memory ceiling ${DEFAULT_MEMORY_CEILING}"
```

Comment:
The logic from xpaas I borrowed this from originally had a higher number (2**60 -1) for the value.  I was afraid of setting heaps that were too large with larger VMs, but I suppose the flip is that "mid range" containers could still consume to much memory.

I'll look into redoing this logic, by always going into the max heap adjustment path, but  capping the max heap we'll set at say 1.5 GiB.  If users want more besides that, then can override via JAVA_OPTS.

Sound OK ?